SimpleWAWS/Models/MakeJenkinsSubscriptionFreeTrialResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace SimpleWAWS.Models
+{
+    public class MakeJenkinsSubscriptionFreeTrialResult
```

Comment:
I have removed this for now. WIll need to revisit this to support multiple resourcegroups in a region